app/src/main/java/fr/free/nrw/commons/contributions/ContributionsPresenter.java

```diff
@@ -77,10 +79,14 @@ public void saveContribution(Contribution contribution) {
             .save(contribution)
             .subscribeOn(ioThreadScheduler)
             .subscribe(() -> {
+                OneTimeWorkRequest updatedUploadRequest = new OneTimeWorkRequest
```

Comment:
Would you mind factorizing the number `10` somewhere (3 occurrences), and adding a comment explaining what is being done? It is OK to have similar comments above each of the 3 occurrences. Thanks!

Reply:
Done :)

Reply:
Great thannks! :-)